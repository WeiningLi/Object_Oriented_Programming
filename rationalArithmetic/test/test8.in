create a 8/4
print a
create b 75/-15
print b
+ a b
- b a
* a b
/ b a
+= a b
-= a b
u- a

