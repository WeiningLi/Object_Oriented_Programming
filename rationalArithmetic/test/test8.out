2
-5
-3
-7
-10
-5/2
-3
2
-2
